tests/test_bytecode_circuit.py

```diff
@@ -69,7 +71,7 @@ def test_bytecode_empty():
 
 def test_bytecode_full():
     bytecodes = [unroll(bytes([7] * (2**k - 1)), randomness)]
-    verify(k, bytecodes, randomness, True)
+    verify(k, bytecodes, randomness, False)  # Last row must be tag=Header
```

Comment:
Maybe a more useful test if the test data is modified with the required last row header so the test passes.

Reply:
Done